content/kubermatic/master/tutorials-howtos/external-clusters/aks/_index.en.md

```diff
@@ -39,29 +51,59 @@ You can click on `Machine Deployments` to get the details:
 
 ## Update Cluster
 
-### Upgrade Version
+### Upgrade Cluster Version
 
 When an upgrade for the cluster is available, a little dropdown arrow will be shown beside the `Control Plane Version` on the cluster’s page.
 To start the upgrade, choose the desired version from the list of available upgrade versions and click on `Change Version`.
 
+![Upgrade Available](/img/kubermatic/master/tutorials/external_clusters/aks_upgrade_available.png "Upgrade Available")
+
 ![Upgrade AKS](/img/kubermatic/master/tutorials/external_clusters/upgrade_aks.png "Upgrade AKS")
 
 If the version upgrade is valid, the cluster state will change to `Reconciling`.
 
-### Scale the Machine Deployment
+## Edit the Machine Deployment
 
-Navigate to the cluster overview, scroll down to machine deployments and click on the edit icon next to the machine deployment you want to edit.
+- Navigate to the cluster overview, scroll down to machine deployments.
+
+- Click on the edit icon next to the machine deployment you want to edit.
 
 ![Update AKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/edit_md.png "Update AKS Machine Deployment")
 
-In the popup dialog, you can now increase or decrease the number of worker nodes that are managed by this machine deployment.
-Either specify the number of desired nodes or use the `+` or `-` to increase or decrease node count.
+- Upgrade Kubernetes Version Select the Kubernetes Version from the dropdown to upgrade the md.
+
+- Scale the replicas In the popup dialog, you can increase or decrease the number of worker nodes that are managed by this machine deployment.
+
+![Update AKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/scale_aks_md.png "Update AKS Machine Deployment")
+
+{{% notice info %}}
+Only one operation can be performed at one point of time. If replica is updated then upgrade kuberntes version will be disabled and vice versa.
+{{% /notice %}}
 
-![Update AKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/update_aks_md.png "Update AKS Machine Deployment")
+## Delete the Node Pool
+
+{{% notice info %}}
+At least one systempool is required in a AKS cluster.
+{{% /notice %}}
+
+Navigate to the cluster overview, scroll down to machine deployments and click on the delete icon next to the machine deployment you want to delete.
+
+![Update AKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/delete_md.png "Delete AKS Machine Deployment")
 
 ## Stopped Cluster State
 
 If the cluster is stopped from the Azure side, you will be able to see the current state of the cluster as stopped.
 Cluster details will not be visible as the details are fetched using kubeconfig, and the kubeconfig is not available for the stopped cluster.
 
 ![AKS Cluster Stopped](/img/kubermatic/master/tutorials/external_clusters/aks_stopped.png "AKS Cluster Stopped")
+
+## Delete Cluster
```

Comment:
**IMHO,**

AKS docs can be aligned with EKS. We can update order so that can be consistent.

**EKS:**
- Delete Cluster
- Delete Node Group

**AKS** 

- Delete Cluster 
- Delete Node Pool
- Stopped Cluster State

**GKE is correct! just like EKS**

wdyt?